### PR TITLE
Added snapshotter deployments files + instuctions to deploy

### PR DESCRIPTION
**Pull request by mickymiek**

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -15,3 +15,32 @@ $ kubectl create -f pod.yaml
 Other helper scripts:
 * `logs.sh` output of the plugin
 * `exec-bash.sh` logs into the plugin's container and runs bash
+
+
+## How to test RBD Snapshot feature
+
+Before continuing, make sure you enabled the required [feature gate](https://kubernetes-csi.github.io/docs/Setup.html#csi-volume-snapshot-support) in your Kubernetes cluster.
+
+In the `examples/rbd` directory you will find four files related to snapshots: `csi-snapshotter-rbac.yaml`, `csi-snapshotter.yaml`, `snapshotclass.yaml` and `snapshot.yaml`.
```
Comment (rootfs) [inline examples/README.md]: make hyperlinks to the yamls

File: examples/rbd/csi-snapshotter-rbac.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    resources: ["volumesnapshotclasses"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["snapshot.storage.k8s.io"]
+    resources: ["volumesnapshotcontents"]
+    verbs: ["create", "get", "list", "watch", "update", "delete"]
+  - apiGroups: ["snapshot.storage.k8s.io"]
+    resources: ["volumesnapshots"]
+    verbs: ["get", "list", "watch", "update"]
+  - apiGroups: ["apiextensions.k8s.io"]
+    resources: ["customresourcedefinitions"]
```
Comment (rootfs) [inline examples/rbd/csi-snapshotter-rbac.yaml]: why do we need crd?

Comment (mickymiek) [reply examples/rbd/csi-snapshotter-rbac.yaml]: It is needed to create `volumesnapshotclasses`, `volumesnapshotcontents` and `volumesnapshots` CRDs if they don't exist. 

https://github.com/kubernetes-csi/external-snapshotter/blob/master/cmd/csi-snapshotter/main.go#L109

File: examples/README.md (hunk 1 above)
Comment (mickymiek) [reply examples/README.md]: @rootfs done

**Pull request merged**